test/test_multithreading.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
```

Comment:
According to https://github.com/msgpack/msgpack-python/issues/613#issuecomment-2914866092, sharing `Packer` objects wouldn't be supported under free-threading, so this test is probably not needed. Could we run the test suite with `pytest-run-parallel` instead?

Reply:
Whatever works: I borrowed this test from another fork, and I'm still learning how to modernize existing `pytest` tests to use `pytest-run-parallel`

Reply:
What I would do is install `pytest-run-parallel` in CI and then add a step in the main test job that looks something like this:

```yml
- name: Install pytest-run-parallel under free-threading
  if: matrix.py == '3.14t-dev' || matrix.py == '3.13t'
  shell: bash
  run: |
    pip install pytest-run-parallel

- name: Run tests in parallel under free-threading
  if: matrix.py == '3.14t-dev' || matrix.py == '3.13t'
  shell: bash
  run: |
    pytest -v --parallel-threads=auto --iterations=20 test
```

Reply:
Great! I think this file can go now, since we're testing with `pytest-run-parallel` and sharing `Packer` objects is not supported.